0028-web-monetization/0028-web-monetization.md

```diff
@@ -40,14 +39,14 @@ This flow refers to the user's agent: in implementation this may be done by an e
 
 - The user visits a webpage.
 - The user's agent sets `document.monetizationState` to `pending`.
-- The user's agent looks for the Web Monetization `<meta>` tags ([specified below](#meta-tags)). The `<meta>` tags MUST be present once `document.readyState` is `interactive`. Implementations MUST NOT process the tags earlier than this, but MAY wait longer before processing.
+- The user's agent looks for the Web Monetization `<meta>` tags ([specified below](#meta-tags)). The `<meta>` tags MUST NOT be inserted dynamically on the client-side.
```

Comment:
Slight nit-pick: Is there any way to insert something dynamically other than by using JavaScript? If not, maybe this should read "dynamically using client-side JavaScript" to be super clear.